scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -130,7 +130,7 @@ class ExperimentalScanner(
         return ortResult.copy(scanner = scannerRun)
     }
 
-    suspend fun scan(packages: Set<Package>, context: ScanContext): Map<Identifier, List<ScanResult>> {
+    fun scan(packages: Set<Package>, context: ScanContext): Map<Identifier, List<ScanResult>> {
```

Comment:
Commit message proposal: "The function does no *need* to be suspendable as it does not call any other `suspend` functions."